variables.tf

```diff
@@ -112,10 +112,12 @@ variable "instance_refresh" {
   type = object({
     strategy = string
     preferences = object({
```

Comment:
```suggestion
    preferences = optional(object({
```

Reply:
That's fine by me.  I will give it a quick test tomorrow and then make the change.

Reply:
I have made the required changes.  Please can you review them.